src/transaction.rs

```diff
@@ -45,6 +52,23 @@ pub struct Contract {
     /// The number of tokens allocated to the `Plan` and any transaction fees.
     pub tokens: i64,
     pub plan: Plan,
+    pub id: Signature,
```

Comment:
Added an id to Contract to support multiple contracts in a single transaction. Before, contracts were identified by in the bank's 'pending' table by transaction signature, which is no longer unique when you can have multiple contracts per transaction